debian/control

```diff
@@ -324,3 +325,11 @@ Description: Tools for Infiniband attached storage (SRP)
  In conjunction with the kernel ib_srp driver, srptools allows you to
  discover and use Infiniband attached storage devices which use the
  SCSI RDMA Protocol (SRP).
+
+Package: python3-pyverbs
+Architecture: linux-any
+Depends: rdma-core (>= 21),
+         python3
+Description: Pyverbs is a Python API over IB verbs. It allows an easy
```

Comment:
The description consists of two parts:
1. The short description one the first line
2. The long description on the second line and following

It looks like you put the beginning of the long description in the first line.

Reply:
done